proto/node/akash/market/v1/types.proto

```
@@ -0,0 +1,35 @@
+syntax = "proto3";
+package akash.market.v1;
+
+import "gogoproto/gogo.proto";
```

Buf (1.55.1)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)